src/Microsoft.Management.UI.Internal/ShowCommand/ViewModel/ModuleViewModel.cs

```diff
@@ -435,21 +435,19 @@ private static bool Matches(WildcardPattern filterPattern, string commandName, s
         /// <returns>Return match result.</returns>
         private static bool MatchesEvenIfInPlural(string commandName, string filter)
         {
-            if (commandName.IndexOf(filter, StringComparison.OrdinalIgnoreCase) != -1)
+            if (commandName.Contains(filter, StringComparison.OrdinalIgnoreCase))
             {
                 return true;
             }
 
             if (filter.Length > 5 && filter.EndsWith("es", StringComparison.OrdinalIgnoreCase))
             {
-                filter = filter.Substring(0, filter.Length - 2);
-                return commandName.IndexOf(filter, StringComparison.OrdinalIgnoreCase) != -1;
+                return commandName.AsSpan().Contains(filter.AsSpan(0, filter.Length - 2), StringComparison.OrdinalIgnoreCase);
```

Comment:
Please use multiple statements for better readability, like
```
var filterSpan = filter.AsSpan(0, filter.Length - 2);
return commandName.AsSpan().Contains(filterSpan, StringComparison.OrdinalIgnoreCase);
```

Reply:
Fixed.